### ISSUE TITLE
Get rid of deprecated Gradle features

### ISSUE THREAD

**Issue by MikeShysh** [JLLeitschuh/ktlint-gradle#60]
Hi, 
ktlint version "3.0.1"
gradle 4.5.1

STR:
run any default check (e.g. :ktlintMainCheck)
Actual result: warning appears:
```
Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.5.1/userguide/command_line_interface.html#sec:command_line_warnings
```
Expected result: no warning
